harness/determined/common/experimental/trial.py

```diff
@@ -176,6 +187,8 @@ def logs(
         ):
             yield log.message
 
+    logs.__signature__ = inspect.signature(iter_logs)
```

Comment:
can this be inside of logs somehow? weird to me that it's just floating in the file.

Reply:
No. It needs to run before `logs` is called. Moved `logs` to be closer, hope that helps.